src/collector-method.cpp

```diff
@@ -612,6 +710,8 @@ class Multi_Collector {
   int ind[INDEX_SIZE];
 
   inline bool have_fields_changed(SEXP field_names, const int& n_fields) const {
+    LOG_DEBUG << n_fields;
```

Comment:
Not sure I would expect `have_fields_changed()` to output the number of fields. Maybe add something like `# fields` before?